usbhostfs_pc/Makefile

```diff
@@ -1,7 +1,7 @@
 OUTPUT=usbhostfs_pc
 OBJS=main.o
 LIBS=-lusb -lpthread
-CFLAGS=-Wall -ggdb -I../usbhostfs -DPC_SIDE -D_FILE_OFFSET_BITS=64 -I. -O2
+CFLAGS=-Wall -ggdb -I../usbhostfs -DPC_SIDE -D_FILE_OFFSET_BITS=64 -I. -O3
```

Comment:
Really not a fan of O3, it is unlikely to have any advantage here.

Reply:
I have reverted the change, thanks